pandokia/common.py

```diff
@@ -488,19 +488,19 @@ def parse_time(arg):
     try:
         d = datetime.datetime.strptime(arg, '%a %b %d %H:%M:%S %Y')
         return d
-    except ValueError:
+    except ValueError as err:
         pass
 
     # just a date
     try:
         d = datetime.datetime.strptime(arg, '%Y-%m-%d')
         return d
-    except ValueError:
+    except ValueError as err:
         pass
 
     # didn't match anything?  ok, well the last exception is
     # as good as anything else we might raise
-    raise
+    raise err
```

Comment:
Not sure this will do what you want, as `err` isn't defined outside of those exception blocks.  Also not sure it's worth gambling that it will and finding out in 2 months say it is still broken.
Safe bet seems to be just `raise ValueError('Did not match anything')` or similar

Reply:
That's fair. Going with that solution...

Reply:
👍